support/ebpf/types.h

```diff
@@ -595,11 +558,11 @@ typedef struct Trace {
   u64 offtime;
 
   // The frames of the stack trace.
-  Frame frames[MAX_FRAME_UNWINDS];
+  u64 frame_data[3072];
```

Comment:
Maybe this value should be moved above to a `#define` with some comments explaining the logic behind the value chosen here. My guess is 3072 = 1024 * 3, as that would been the size of the frame buffer had #908 landed, so his means "Room for 1024 frames, each with 3 x 64 bit values", as the frame struct was previously 3 x64 bit values.